sdk/formrecognizer/Azure.AI.FormRecognizer/tests/samples/Sample9_DifferentiateOutputModelsTrainedWithAndWithoutLabels.cs

```diff
@@ -0,0 +1,118 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Azure.AI.FormRecognizer.Models;
+using Azure.AI.FormRecognizer.Tests;
+using Azure.AI.FormRecognizer.Training;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.AI.FormRecognizer.Samples
+{
+    public partial class FormRecognizerSamples : SamplesBase<FormRecognizerTestEnvironment>
+    {
+        /// This sample demonstrates the differences in output that arise when StartRecognizeCustomForms
+        /// is called with custom models trained with labels and without labels.
+
+        [Test]
+        public async Task OutputModelsTrainedWithLabels()
+        {
+            string endpoint = TestEnvironment.Endpoint;
+            string apiKey = TestEnvironment.ApiKey;
+            string trainingFileUrl = TestEnvironment.BlobContainerSasUrl;
+            string formFilePath = FormRecognizerTestEnvironment.CreatePath("Form_1.jpg");
+
+            FormTrainingClient trainingClient = new FormTrainingClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+            FormRecognizerClient client = new FormRecognizerClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+
+            // Model trained with labels
+            CustomFormModel modelWithLabels = await trainingClient.StartTraining(new Uri(trainingFileUrl), useTrainingLabels: true).WaitForCompletionAsync();
```

Comment:
nit: I'd be verbose and call this modelTrainedWithLabels just to be pedantic.  If you hate it, I can be flexible.

Reply:
because this is a sample, I am ok with long names :P